t/24-worker-settings.t

```diff
@@ -61,8 +61,8 @@ is_deeply(
 
 subtest 'apply settings to app' => sub {
     my ($setup_log_called, $setup_log_app);
-    my $mock = Test::MockModule->new('OpenQA::Setup');
-    $mock->mock(
+    my $mock = Test::MockModule->new('OpenQA::Worker::Settings');
```

Comment:
Shouldn't this mock the `OpenQA::Log` module now?

Reply:
The import is `OpenQA::Log 'setup_log'` which means the function is logically in the importing module which is Settings.pm.

Reply:
Ok, considering that the tests pass you're likely right.